katdal/sensordata.py

```diff
@@ -296,11 +296,11 @@ def root(self):
         return TelstateToStr(self._telstate.root())
 
     def keys(self, filter='*'):
-        return to_str(self._telstate.keys(filter))
+        return self._telstate.keys(filter)
 
     @property
     def prefixes(self):
-        return to_str(self._telstate.prefixes)
+        return self._telstate.prefixes
```

Comment:
I think this function can also safely be deleted.

Reply:
Yip.